### ISSUE TITLE
Abnormal fetal morphology 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7210]
**Preferred term label:**
Abnormal fetal morphology

Should this be a new term underneath [Abnormality of prenatal development or birth HP:0001197](https://hpo.jax.org/app/browse/term/HP:0001197), which currently has a mix of different categories as its children?

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
